roles/tpa_single_node/vars/main.yml

```
@@ -12,52 +12,51 @@ tpa_single_node_systemd_directory: /etc/systemd/system
 tpa_single_node_default_empty: ""
 
 # DB_SERVICE
-tpa_single_node_pg_host: "{{ lookup('env', 'TPA_PG_HOST' default='192.168.121.60') }}"
+tpa_single_node_pg_host: 192.168.121.60
```

The lookup is good idea meanwhile there is a typo.
```suggestion
tpa_single_node_pg_host: "{{ lookup('env', 'TPA_PG_HOST') | default('192.168.121.60') }}"
```